Inferential Stats/human_temp/sliderule_dsi_inferential_statistics_exercise_1_Michael_Skrzypiec.ipynb

```diff
@@ -0,0 +1,794 @@
2o8/krBrN5a3bqZ6\n2nVK8T9qEUJtzlGNUCpxEWeuUo4EklZ0t7JSTTlFi4KDQ9KtRAnT+8TtwoVVJbG1taq1iY1Vt+ho\n9VEYEaE+U69cgcuX1fe5S5fUflxc+m/N0dE0gUm8L1tWOh1nhm4Jy88//8zEiRPZl6wdwtvbm//7\nv//j9ddfN5Z98cUXhIeHM336dADu3r1Lo0aNOHToECWe41t8XkxYYmPVH9CZM2r9j9On4bejGv+c\nf4Aj4ZTmOqW5Thn+oZzFdeqW+YfqRa9R6uFFLK9cTn9WSmtrtaKhp6e6NW6s/sqEECK7aJqajTI4\nOOmWOC1CGhKsrIkrVxGtkjOWzhWxLFMKQ+lSUCrZ7eWXwdb2hYYaHw///KOmlzp7VoWZeH/jRvqP\nLVIkKXmpXl3VbpcvD+XKQenS0gk5kW6XITQ0FGdnZ5MyJycnQkNDU5zn5uZm3C9evDjFihUjNDT0\nuRIW3WmaWsr4Se8u7XEM8VExxEXFEvco6T7qXgxRkbFER8YQHRlD7N2HxtvjOw+JufuQuHsPib/3\nEB4+pCj3cCSCt7hDLyIowR2sSSXtTwCuphKXhYX6a6lcWU1l6+ambtWrq2YfIYTIKQZDUttK796q\nLDJS1f6GhCTdLlxQNTOARVwsNpfOw6XzsC+d57a1VfM/FStmep+4XbiwOsfOLuV98u0CBcDaGktL\nS8pbWVG+qhXNa1qpqmsrK7CyIiLSkrPnrTj7lyVnzhqMiczlyyqU+/dVi/uRIynDtLBQSUv58uq+\nRAnVh8be3nS7UKGk0JLfChRQoVha5v5aHN0SlqioKJOmHwA7Ozuio6OzdF56IiIiuPvUYlz//PMP\nADcySn2zw+PH0L49/PVXhqcagIJPbhl66qcZC9x8+kCBAurbxUsvqfuSJdXoncTeY+XKqdqUpyUf\nNiSEEHr6z3/UrV27pLKoKNUXJvF2+TJcv65GMSbekk+1GxenOvQlnyMqO0N+cmudmD1YWqIVs0DT\nDMQnGIhPgATNQFy8gYQEAxrJbjcNcBM0QMMAPHX8yS0GeIwB9d/O9FhyBsBgAcWKQ7HiFvDee2r2\n8WxWqlQprJ6juki3hCWt5KRgQdN/zba2tpk6Lz2rV6/G398/1WPdunXL9PO8cJUq6ffad+6o29mz\n+sUghBA5wdZWVVGI1K1cqW7ZbNOmTdSsWTPLj9ctYalUqRJr1qwxKQsLC6N9+/YmZc7OzoSFhRn3\n79y5Q2RkZIrmpPR0796ddskzcVRT06BBgwgICKBixYrP/gbymCtXrtCzZ09WrVpFefnDluvxFLke\npuR6mJLrYUquh6nE61GgQIHneh7dEpaGDRsSExPDmjVr8PX1ZfPmzdy5cwfPp+Z7b9euHT169KBT\np07UrFmT2bNn07RpU4oVK5bp17K3t8fe3j7VY2XLls0znW6fR+yTqtJSpUrJ9UCux9PkepiS62FK\nrocpuR6mEq+HpaVlBmemT7fFD21sbFi2bBlbt27Fw8ODtWvXsmjRImxtbZkwYQITJ04EoFq1akyZ\nMoWPP/6Yxo0bEx4ezmeffaZX2EIIIYTQga6DpapUqcK6detSlE+aNMlk//XXXzcZ6iyEEEKI/EW3\nGhYhhBBCiMyynJjY9pIP2dra0qBBgxTDpvMruR6m5HqYkuthSq6HKbkepuR6mHoR1yPfLn4ohBBC\niNxDmoSEEEIIYfYkYRFCCCGE2ZOERQghhBBmTxIWIYQQQpg9SViEEEIIYfYkYRFCCCGE2ZOERQgh\nhBBmTxIWIYQQQpi9fJGwBAcH89Zbb1G3bl06d+5MSEgIAKdOnaJGjRq4u7vj5uaGu7s7S5cu1Tna\n7JfW9YiNjWXKlCk0bNiQhg0bMnbsWOMqm3lZatfj+vXrxt+JxJuLi0u+WNMqrd+Pf//9lwEDBtCg\nQQOaNGnCnDlzdI40Z6R1Pa5cuULfvn2pX78+rVu3ZvPmzTpHmr1CQkJo0qSJcT8yMpLBgwdTr149\nvLy8CAwMNDl/1qxZNGrUCA8PDz777DPy2hylz3o9ADRNY8iQIaxZsyYnQ80Rz3I9YmNjmTp1Kp6e\nnnh4eDBw4ECuX7+e8YtoedzVq1e1OnXqaBs3btTi4+O1/fv3aw0aNNDCw8O1DRs2aP3799c7xByV\n3vWYNm2a5ufnp0VGRmr37t3TfH19tSVLlugdcrZK73okd+vWLa1JkybawYMHdYo0Z6R2PTw8PLRb\nt25pQ4YM0aZNm6YlJCRoN27c0Fq0aKFt3rxZ75CzVVrX48aNG1q7du20Tz75RHv8+LEWFhamNW/e\nXDtw4IDeIWeLjRs3avXq1dMaNmxoLBsyZIg2cuRILSYmRjt58qTWoEED7eTJk5qmado333yjtW/f\nXgsPD9fCw8O1jh07asuXL9cr/BfuWa+Hpqnfpb59+2rVqlXTVq9erUfY2eZZr8eXX36pdenSRbt9\n+7b2+PFjbfz48VqXLl0yfJ08X8Py888/U7VqVXx8fLCwsKBZs2bUqVOHnTt3cubMGapXr653iDkq\nveuxYcMGxo0bR5EiRShatCjz58/H29tb75CzVWrXw9XVlV27dpmcN378eNq0aUPjxo11ijRnpHc9\nLl26RFxcHHFxcWiahqWlJba2tnqHnK3Suh579uwhNDSUcePGYWNjQ8WKFenatWuq36pzu8WLF7N6\n9WoGDhxoLHv06BFBQUG8//77WFtbU7t2bby9vY21TFu2bMHPzw8HBwccHBzo378/mzZt0ustvFBZ\nuR6xsbF07NiRatWq4ebmplfo2SIr1yM6OppBgwZRokQJbGxs6Natm7HmMj15PmFJSEhI8aFqMBi4\ndOkSZ8+e5fjx47Ro0QIvLy9mzJiR55tA0roeP/30EwkJCZw8eZLWrVvTrFkzVq5cyUsvvaRTpDkj\ntethYWHBpUuXjPvBwc
```

Comment:
Use lower case + underscores for function names https://www.python.org/dev/peps/pep-0008/#function-names